deluge/core/core.py

```diff
@@ -159,20 +159,20 @@ def __init__(self, listen_interface=None, outgoing_interface=None, read_only_con
         # store the one in the config so we can restore it on shutdown
         self._old_listen_interface = None
         if listen_interface:
-            if deluge.common.is_ip(listen_interface):
+            if (windows_check() and deluge.common.is_ip(listen_interface)) or not windows_check():
                 self._old_listen_interface = self.config['listen_interface']
                 self.config['listen_interface'] = listen_interface
             else:
-                log.error('Invalid listen interface (must be IP Address): %s', listen_interface)
+                log.error('Invalid listen interface (must be adapter name or IP Address on unix ip on win): %s', listen_interface)
```

Comment:
Perhaps it's clearer to say:
`Invalid listen interface. (Must be IP Address or, on Linux, adapter name.)`? (That question mark is because I don't actually know if it's clearer.)

Alternatively, just add a comma, (and fix capitalization,) into your current message. i.e. `must be adapter name or IP address on Linux, or IP address on Windows`